gst-libs/mfx/gstmfxsurface_vaapi.c

```diff
@@ -190,6 +192,115 @@ gst_mfx_surface_vaapi_init (GstMfxSurfaceVaapi * surface)
 {
 }
 
+GstMfxSurface *
+gst_mfx_surface_vaapi_new_external (GstMfxContext * context, const GstVideoInfo * info)
+{
+  GstMfxSurfaceVaapi *surface;
+
+  g_return_val_if_fail (context != NULL, NULL);
+  g_return_val_if_fail (info != NULL, NULL);
+
+  surface = g_object_new (GST_TYPE_MFX_SURFACE_VAAPI, NULL);
+  if (!surface)
+    return NULL;
+
+  return
+      gst_mfx_surface_new_external (GST_MFX_SURFACE (surface),
+          context, info, NULL);
+}
+
+GstMfxSurface *
+gst_mfx_surface_new_from_buffer_proxy (GstMfxContext * context, GstMfxPrimeBufferProxy * proxy, const GstVideoInfo * info)
```

Comment:
gst_mfx_surface_vaapi_new_from_buffer_proxy() to reflect the naming standards.

Reply:
Agreed. Siew Hoon appreciate it if you can make the change.